rnd/autogpt_libs/autogpt_libs/auth/models.py

```diff
@@ -0,0 +1,19 @@
+from dataclasses import dataclass
+
+
+# Using dataclass here to avoid adding dependency on pydantic
+@dataclass(frozen=True)
+class User:
```

Comment:
I've created a user object directly from the data found in the jwt token so we can use this in our routes